packages/components/src/view/component.js

```diff
@@ -19,8 +19,12 @@ import styled from '@emotion/styled';
  * 	);
  * }
  * ```
+ *
+ * @type {import('../ui/context').PolymorphicComponent<'div', {}>}
  */
+// @ts-ignore
```

Comment:
Wondering if there's a way of achieving the cast to `PolymorphicComponent` without adding `// @ts-ignore`

Reply:
There isn't, we're casting to two completely unrelated types, and the `selector` won't exist on the value returned by ``` styled.div`` ```

Reply:
Gotcha. Let's keep the `// @ts-ignore`, then. Thank you for the explanation!